.github/workflows/pr.yml

```diff
@@ -0,0 +1,37 @@
+name: PR
+
+on:
+  pull_request:
+    branches: [ "main", "master" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  initialise:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+          
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
```

Comment:
we need to push to quaq.io.

I think we can use `registry: quay.io` according to https://medium.com/@erdal.genc09/github-action-to-automate-build-push-for-dockerfiles-repository-to-quay-io-docker-registry-9ebbf6007e2

What I don't understand is ... I only have a `OAUTH_TOKEN` not a username. But I guess I can try a few option and see if that works.